### PR TITLE
[NETID] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/win32/netid/lang/ro-RO.rc [hunk 1]
```diff
@@ -2,93 +2,93 @@
...
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_COMPNAME, IDC_STATIC, 4, 3, 26, 24, SS_ICON
-    LTEXT "Informațiile de mai jos vor fi utilizate pentru identificarea calculatorului în rețea.", IDC_STATIC, 40, 5, 204, 20
-    LTEXT "Descrierea calculatorului:", IDC_STATIC, 7, 40, 90, 9
+    LTEXT "Pentru identificarea computerului în reţea, ReactOS utilizează următoarele informaţii.", IDC_STATIC, 40, 5, 204, 20
+    LTEXT "&Descriere computer:", IDC_STATIC, 7, 40, 90, 9
     EDITTEXT IDC_COMPDESC, 98, 38, 146, 13, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Numele complet:", IDC_STATIC, 7, 68, 85, 9
+    LTEXT "Numele complet al computerului:", IDC_STATIC, 7, 68, 85, 9
```
Comment (JoachimHenze) [inline dll/win32/netid/lang/ro-RO.rc:19]: looks too long to my eyes for 85 width

File: dll/win32/netid/lang/ro-RO.rc [hunk 2]
```diff
@@ -2,93 +2,93 @@
...
     EDITTEXT IDC_COMPUTERNAME, 98, 68, 144, 12, ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Grup de lucru:", IDC_WORKGROUPDOMAIN, 7, 84, 64, 9
+    LTEXT "Domeniu:", IDC_WORKGROUPDOMAIN, 7, 84, 64, 9
     EDITTEXT IDC_WORKGROUPDOMAIN_NAME, 98, 84, 144, 12, ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Utilizați expertul de identificare al rețelei (apăsând „ID de rețea”) pentru a crea un utilizator și a deveni membru al unui domeniu de rețea.", IDC_STATIC, 7, 113, 172, 24
-    PUSHBUTTON  "I&D de rețea…", IDC_NETWORK_ID, 190, 114, 58, 15
-    LTEXT "Pentru a schimba identitatea calculatorului sau a intra într-un domeniu, apăsați „Schimbare”.", IDC_STATIC, 7, 149, 170, 17
-    PUSHBUTTON  "S&chimbare…", IDC_NETWORK_PROPERTY, 190, 149, 58, 15
+    LTEXT "Pentru a utiliza Expertul de identificare în reţea pentru asocierea la un domeniu şi crearea unui cont local al utilizatorului, faceţi clic pe IDentificator reţea.", IDC_STATIC, 7, 113, 172, 24
+    PUSHBUTTON  "&IDentificator reţea", IDC_NETWORK_ID, 190, 114, 58, 15
```
Comment (JoachimHenze) [inline dll/win32/netid/lang/ro-RO.rc:24]: looks too long to my eyes for 58 width. If it overflows, I do recommend to shorten the string.

File: dll/win32/netid/lang/ro-RO.rc [hunk 3]
```diff
@@ -2,93 +2,93 @@
...
+    8 "Numele de domeniu ""%1"" nu este formatat corespunzător.  Punctele (.) sunt utilizate pentru a separa domeniile.  Fiecare domeniu este limitat la %2!d! caractere. Exemplu: domeniul-1.reactos.org."
+    10 "Noul nume al computerului ""%1"" este prea lung. Numele nu poate fi mai lung de 63 de caractere."
+    11 "Numele nou introdus pentru computer nu este formatat corespunzător. Numele standard pot conţine litere (a-z, A-Z), cifre (0-9) şi cratime (-), nu şi spaţii sau puncte (.). Numele nu poate fi compus în întregime numai din cifre."
+    22 "Bun venit la grupul de lucru ""%1""."
+    23 "Bun venit la domeniul ""%1""."
+    24 "Pentru ca modificările să aibă efect reporniţi computerul."
+    25 "Aveţi posibilitatea să modificaţi numele şi calitatea de membru al acestui computer. Modificările pot afecta accesul la resursele reţelei."
+    1021 "Notă: Numai administratorii pot modifica elementele de identificare pentru acest computer."
+    1022 "Notă: Imposibil de modificat elementele de identificare pentru acest computer, deoarece:"
+    1029 "Numele nou ""%1"" al computerului este un număr.  Numele nu poate fi un număr."
```
Comment (JoachimHenze) [inline dll/win32/netid/lang/ro-RO.rc:86]:
```suggestion
    1029 "Numele nou ""%1"" al computerului este un număr. Numele nu poate fi un număr."
```

File: dll/win32/netid/lang/ro-RO.rc [hunk 4]
```diff
@@ -2,93 +2,93 @@
...
+    1030 "Numele nou ""%1"" al computerului conţine caractere nepermise. Caracterele care nu sunt permise includ ` ~ ! @ # $ % ^ & * ( ) = + [ ] { } \\ | ; : ' \" , < > / şi ?"
```
Comment (JoachimHenze) [inline dll/win32/netid/lang/ro-RO.rc:87]: The MSVC compiler does not compile due to
`D:\a\reactos\reactos\src\dll\win32\netid\lang/ro-RO.rc(87) : error RC2148 : expected numeric constant in string table`
I do hope that my suggestion helps (but have not tested it):
```suggestion
    1030 "Numele nou ""%1"" al computerului conţine caractere nepermise. Caracterele care nu sunt permise includ ` ~ ! @ # $ % ^ & * ( ) = + [ ] { } \\ | ; : ' "" , < > / şi ?"
```

**Review by JoachimHenze — CHANGES_REQUESTED** [review]

File: dll/win32/netid/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc:19]: I checked with ReactOS's latest nightly build LiveCD (reactos-livecd-0.4.15-dev-8038-g28eb4e7-x86-gcc-lin-dbg) and it fits perfectly. I also checked with XP's dll and it also fits in the box. Here is the proof.

2024-05-05 23_26_45-Resource Hacker - netid dll mui

File: dll/win32/netid/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc:24]: I checked with ReactOS's latest nightly build LiveCD (reactos-livecd-0.4.15-dev-8038-g28eb4e7-x86-gcc-lin-dbg) and it fits exactly in the box. I also checked with XP's dll and it also fits in the box. Here is the proof.

ReactOS latest nightly build
2024-05-05 23_32_49-Resource Hacker - netid dll

XP/2k3
2024-05-05 23_33_09-Resource Hacker - netid dll mui

File: dll/win32/netid/lang/ro-RO.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/win32/netid/lang/ro-RO.rc:19]: Thx. Resolved.

File: dll/win32/netid/lang/ro-RO.rc (hunk 2 above)
Comment (JoachimHenze) [reply dll/win32/netid/lang/ro-RO.rc:24]: Thx. Resolved.

File: dll/win32/netid/lang/ro-RO.rc [hunk 5]
```diff
@@ -2,93 +2,93 @@
...
-    1021 "Notă: Doar administratorii pot schimba identitatea calculatorului."
-    1022 "Notă: Identitatea calculatorului nu poate fi schimbată deoarece:"
-    1029 "Noul nume de calculator ""%1"" e un număr. Numele nu poate fi un număr."
-    1030 "Noul nume „%1” conține caractere nepermise. Caracterele nepermise sunt ` ~ ! @ # $ %% ^ & * ( ) = + [ ] { } \\ | ; : ' "" , < > / și ?"
-    1031 "Primul domeniu al numelui de domeniu ""%1"" este un număr. Primul domeniu nu poate fi un număr."
-    1032 "Numele de domeniu ""%1"" conține caractere ce nu sunt permise. Numele standard de DNS pot conține litere (a-z, A-Z), numere (0-9), și cratime, dar nu spații. Punctele (.) sunt folosite la domeniile separate. De exemplu: domeniu-1.reactos.org."
+    7 "Numele domeniului ""%1"" nu este conform specificaţiilor serviciului Internet Domain Name, deşi este un nume ReactOS legal. Pentru nume nestandard utilizaţi serverul DNS ReactOS."
+    8 "Numele de domeniu ""%1"" nu este formatat corespunzător. Punctele (.) sunt utilizate pentru a separa domeniile.  Fiecare domeniu este limitat la %2!d! caractere. Exemplu: domeniul-1.reactos.org."
```
Comment (JoachimHenze) [inline dll/win32/netid/lang/ro-RO.rc:77]:
```suggestion
    8 "Numele de domeniu ""%1"" nu este formatat corespunzător. Punctele (.) sunt utilizate pentru a separa domeniile. Fiecare domeniu este limitat la %2!d! caractere. Exemplu: domeniul-1.reactos.org."
```
This suggestion can just be taken as is.
One question: There was no %2 in here before. But now there is. Does it exist in en-US as well?

File: dll/win32/netid/lang/ro-RO.rc (hunk 2 above)
Comment (JoachimHenze) [inline dll/win32/netid/lang/ro-RO.rc:24]:
```suggestion
    PUSHBUTTON  "&Identificator reţea", IDC_NETWORK_ID, 190, 114, 58, 15
```
better?

**Comment by AndreiMiloiu** [comment]
@JoachimHenze I will review the mentions and answer you tomorrow.

File: dll/win32/netid/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc:77]: No, it isn't in ReactOS. I accidentally added it. I'll fix it.

File: dll/win32/netid/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc:24]: I don't think so. I refers to Network ID. This is why MSDN chose this way. Here is the proof.

Comment (JoachimHenze) [reply dll/win32/netid/lang/ro-RO.rc:24]: No this absolutely does not convince me. The ros button is smaller than the MS button. The old master head state "&ID de reţea" does fit the button fine as we can see in your pic. But you now changed that to "&IDentificator reţea" abd that would definitely overflow the button in ros. Do not extend that strings length please.

Comment (JoachimHenze) [reply dll/win32/netid/lang/ro-RO.rc:24]: Ok, resolved.

Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc:24]: I understand. In this way I will keep the accelerator, but I'll revert the change.

Comment (JoachimHenze) [reply dll/win32/netid/lang/ro-RO.rc]: Ok, resolved.

File: dll/win32/netid/lang/ro-RO.rc (hunk 5 above)
Comment (JoachimHenze) [reply dll/win32/netid/lang/ro-RO.rc]: Resolved.

**Pull request merged**